root/etc/e-smith/templates/etc/sudoers.d/50_servermanager_perms/90SuperAdministrator

```diff
@@ -0,0 +1,48 @@
+{
+    use File::Find;
+    use esmith::ConfigDB;
+    my $db = esmith::ConfigDB->open_ro();
+    my $groupAdmins = $db->get_prop('admins','group') || 'domain admins';
+
+    (my $groupAdminsAlias = uc $groupAdmins) =~ s/ //;
+    (my $groupAdminsSudo = $groupAdmins) =~ s/ /\\ /;
+
+    $OUT .= qq (
+#
+# default group administrator $groupAdmins
+#
+
+%$groupAdminsSudo ALL=NOPASSWD: NSAPI_SUPER_ADMINISTRATOR
+# server-manager does not require a tty
+Defaults:%$groupAdminsSudo !requiretty
+);
+
+    # here we list all API and enable it for administrators
+    my @files;
+    my @dirpath=qw(/usr/libexec/nethserver/api);
+    
+    find(sub {
```

Comment:
API for application must be declared by the application developer and not calculated by this code. It's the app developer's responsibility: let's write a docs guide about it!